README.md

```diff
@@ -78,26 +78,45 @@ This module exports a single function.
 
 ```js
 const plugin = createPlugin(
-  globals: Object,
+  globals: Object | Function,
   {
     include?: Array,
-    exclude?: Array
+    exclude?: Array,
+    dynamicWrapper?: String | Function
```

Comment:
I suggest removing the ability to use a string as `dynamicWrapper`:

1. Reduce complexity.
2. Using a function is way more explicit than using a string. Compare
    ```js
    dynamicWrapper: "Promise.resolve"
    ```
    vs
    ```js
    dynamicWrapper: name => `Promise.resolve(${name})`
    ```
3. Better forward compatibility. In the future, we may want to pass `moduleId` to the wrapper, which won't be compatible with a simple string.

Reply:
I don't see any downsides to supporting both, but it's trivial enough to re-add string support if needed. Updated